libcontainer/standard_init_linux.go

```diff
@@ -150,6 +152,18 @@ func (l *linuxStandardInit) Init() error {
 			return err
 		}
 	}
-
-	return system.Execv(l.config.Args[0], l.config.Args[0:], os.Environ())
+	// check for the arg before waiting to make sure it exists and it is returned
+	// as a create time error
+	name, err := exec.LookPath(l.config.Args[0])
```

Comment:
If folks intend to never call `start` (which seems fine to me for some use-cases), then we probably don't want to require a `process` config at all.  If they do intend to call `start`, whether the command exists at `execve`-time seems different enough from whether the command exists at `create` time for folks to want this early check to at least be optional (e.g. “don't worry, by the time I call `start` I'll have created that command”).